### DISCUSSION TITLE
error in decoding

### DISCUSSION THREAD

**Discussion by zzhiyuan59** [py-pdf/pypdf#1075] [Q&A]
UnicodeDecodeError: 'utf-16-be' codec can't decode byte 0xc5 in position 0: 
<img width="1072" alt="image" src="https://user-images.githubusercontent.com/25758589/177973836-f81442e6-a271-4f90-884c-731ed8ceaffe.png">
<img width="872" alt="image" src="https://user-images.githubusercontent.com/25758589/177973895-678106c0-7a93-46b7-9f03-866c980d7d1b.png">

**Comment by pubpub-zz** [comment]
@zzhiyuan59 ,
can you provide the pdf file with the issue

**Reply by zzhiyuan59** [reply]
the file is from the Comprehensive LATEX Symbol, the URL is https://tug.ctan.org/info/symbols/comprehensive/symbols-a4.pdf.

and those page cannot be read properly: 1, 6, 9, 10, 13, 14, 15, 16, 18, 19, 21, 24, 25, 27, 28, 30, 31, 36, 39, 51, 53, 61, 62, 73, 88, 89, 91, 94, 95, 97, 100, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 119, 124, 125, 126, 127, 129, 130, 133, 134, 157, 161, 163, 179, 190, 196, 199, 236, 255, 256, 257, 259, 260, 261, 262, 263, 264, 265, 266, 267, 268, 269, 270, 271, 272, 273, 274, 276, 277, 278, 279, 281, 282, 283, 284, 286, 288, 291, 293, 294, 296, 297, 298, 299, 300, 302, 303, 307, 310, 311, 312, 313, 314, 315, 316, 317, 318, 319, 320, 327, 328, 330, 331, 332, 333, 334, 337, 339, 340, 341, 342, 343, 345, 346, 347, 368, 369, 371, 372, 373, 374, 375, 376, 377, 378, 379, 380, 382, 383, 384, 385, 386, 387, 388, 389, 405, 406, 407, 415, 416, 417, 418, 420, 421

**Reply by pubpub-zz** [reply]
I'm finalizing a fix

**Reply by zzhiyuan59** [reply]
oh man, you are amazing! Thank you very much!

**Comment by pubpub-zz** [comment]
fixed in PR #1084
